app/share/views.py

```diff
@@ -88,6 +96,332 @@ def resources():
 @share.route('/files/<model_run_uuid>')
 @login_required
 def files(model_run_uuid):
+
+    if not os.path.exists(app.config['UPLOAD_FOLDER']):
+        os.makedirs(app.config['UPLOAD_FOLDER'])
+
     "View of file submission for as yet unselected resource to add to"
     model_run_uuid = model_run_uuid
     return render_template('share/files.html', model_run_uuid=model_run_uuid)
+
```

Comment:
When the user selects 'Add files to this resource', they should be directed here, but here they should only be prompted to upload a file. This could be used for any type of file, not just PRMS, so it doesn't make sense to have two places to choose a file to upload. Here are the requirements for this page:

* a single box/location where a user clicks 'Browse' to select the file they want to upload
* a dropdown list containing three items, which will go in the "model_name". The three options are "prms", "isnobal", and "reference"
* any other metadata that is necessary to fill in `metadata_from_file`